src/tools/spin/spinquic.cpp

```diff
@@ -1364,13 +1369,9 @@ CXPLAT_THREAD_CALLBACK(RunThread, Context)
     do {
         Gbs Gb;
 
-#ifdef QUIC_BUILD_STATIC
```

Comment:
We don't want to use a lock with dynamic linking, because it must support parallel callers. We can't crash if two things call into the dll/so in the same process.

Reply:
How things are different for static linking in this case? If we support parallel callers to open and close msquic for dynamic linking, static linked msquic should work too, right?